rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -126,7 +121,6 @@ spec:
               Tekton results on cluster {{ $labels.source_cluster }} only has an API success rate of {{ $value | humanizePercentage }}.
             alert_team_handle: <!subteam^S04PYECHCCU>
             team: pipelines
-            # link will be available once https://issues.redhat.com/browse/SRVKP-5900 is done
             runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/sre/tekton-result-high-errors.md
```

Comment:
Should this still be the old link?

Reply:
No, this is the correct link. We don't need to change this.